docs/multiarch.rst

```diff
@@ -324,6 +324,122 @@ platforms will continue. Once all have either succeeded or failed, the
 orchestrator build will fail. No content will be available from the
 registry.
 
+Logging
+-------
+
+Logs from worker builds will be made available via the orchestrator
+build, and clients (including koji-containerbuild) will be able to
+separate individual worker build logs out from that log stream using
+a new osbs-client API method.
+
+Multiplexing
+~~~~~~~~~~~~
+
+In order to allow the client to de-multiplex logs containing a mixture
+of logs from an orchestrator build and from its worker builds, a new
+logging field, platform, is used. Within atomic-reactor all logging
+should be done through a LoggerAdapter which adds this 'platform'
+keyword to the 'extra' dict passed into logging calls. These objects
+should come from a factory function::
+
+  def get_logger(name, platform=None):
+      return logging.LoggerAdapter(logging.getLogger(name),
+                                   {'platform': platform or '-'})
+
+The logging format will include this new field::
+
+  %(asctime)s platform:%(platform)s - %(name)s - %(levelname)s - %(message)s
+
+resulting in log output like this::
+
+  2017-06-23 17:18:41,791 platform:- - atomic_reactor.foo - DEBUG - this is from the orchestrator build
+  2017-06-23 17:18:41,791 platform:x86_64 - atomic_reactor.foo -  DEBUG - text:2017-06-23 17:18:41,400 platform:- atomic_reactor.foo -  DEBUG - this is from a worker build
```

Comment:
Clearer now. Although I think it might be worth mentioning that for lines that have been proxied from a worker build, the wrapping orchestrator log message is dropped. For instance, with this line:
```
2017-06-23 17:18:41,791 platform:x86_64 - atomic_reactor.foo - INFO - 2017-06-23 17:18:41,400 platform:- atomic_reactor.foo -  DEBUG - this is from a worker build
```
This whole prefix is removed:
```
2017-06-23 17:18:41,791 platform:x86_64 - atomic_reactor.foo - INFO - 
```

Resulting in this:
```
2017-06-23 17:18:41,400 platform:- atomic_reactor.foo -  DEBUG - this is from a worker build
```